apps/indexer/ponder.config.ts

```diff
@@ -34,6 +41,10 @@ const chains = Object.entries(env.CHAIN_CONFIGS).reduce(
         env.CHAIN_ANVIL_ECP_COMMENT_MANAGER_ADDRESS_OVERRIDE
           ? env.CHAIN_ANVIL_ECP_COMMENT_MANAGER_ADDRESS_OVERRIDE
           : SUPPORTED_CHAINS[chainId].commentManagerAddress,
+      publicClient: createPublicClient({
```

Comment:
why is this needed now but not needed previously?

Reply:
makes it easier to manage clients than instantiaing them all over the place.